src/gpu_util.cpp

```diff
@@ -34,6 +34,10 @@
 
 #include "rocm_smi/rocm_smi.h"
 #include "include/gpu_util.h"
+#define __HIP_PLATFORM_HCC__
```

Comment:
This is deprecated we should ne using HIP_PLATFORM_AMD

Reply:
__HIP_PLATFORM_HCC__ is present in other code as well. Need to make this change separately.